### DISCUSSION TITLE
EntityMetadataNotFoundError: No metadata for "CategoryEntity" was found.

### DISCUSSION THREAD

**Discussion by golubvladimir** [tada5hi/typeorm-extension#386] [Q&A]
Hello, I make my pet project. I want make seeding.

https://github.com/golubvladimir/pizza-app/tree/backend/add-seeding

Now I have error: 
`EntityMetadataNotFoundError: No metadata for "CategoryEntity" was found.`

How to fix it?

**Comment by golubvladimir** [comment]
I fixed paths. The error is gone. But, seeders and factories not call.

**Comment by golubvladimir** [comment]
I fixed paths. The error is back.

**Comment by tbm5k** [comment]
Hey, in your data source configuration, you have this line:
```js
entities: [
          `${ __dirname }/**/*.entity.js`
        ],
``` 
``` __dirname ``` prints out the directory to the folder in which the function is running therefore, a better approach would be to put all your entities under one folder and use the code below
```js
entities: [
          `${process.cwd()}/src/entities/*.ts`
        ]
```
